contentheader/README.md

```diff
@@ -0,0 +1,96 @@
+# Main Widget Content Header
+
+> Insert a widget into the `contentHeader` section of a `MainAreaWidget`—useful for toolbars, headers, etc.
+
+![Demo: activate the extension using Command Palette to populate the time in GMT in the content header](./preview.gif)
+
+This JupyterLab example extension is intended to demo one specific feature of `MainAreaWidget`, namely, its `contentHeader` section.
+
+> As background, `MainAreaWidget` is a high-level JupyterLab widget that conventionally is used to enclose the Launcher (shown above) or a notebook editor. The `contentHeader`, in turn, is a Lumino `BoxPanel` widget positioned at the very top of this main area. This makes the `contentHeader` potentially useful to extensions needing some place to put content that the user will _always see_.
+
+In code: after we get a `MainAreaWidget`, for example via
+
+```ts
+// src/index.ts#L37-L37
+
+const main = app.shell.currentWidget;
+```
+
+we can then create a widget of interest, for example as
+
+```ts
+// src/index.ts#L40-L40
+
+const widget = new Widget();
+```
+
+before finally adding it to the JupyterLab main area's `contentHeader` real estate at its very top:
+
+```ts
+// src/index.ts#L45-L45
+
+main.contentHeader.addWidget(widget);
+```
+
+## Install
+
+First, ensure you've followed the installation instructions in the [top-level README](../README.md), e.g.:
+
+```bash
+# clone the repository
+git clone https://github.com/jupyterlab/extension-examples.git jupyterlab-extension-examples
+
+# go to the extension examples folder
+cd jupyterlab-extension-examples
+
+# create a new environment
+conda env create
+
+# activate the environment
+conda activate jupyterlab-extension-examples
+```
+
+Then build this extension and launch it—this largely also follows from the instructions in the top-level [README](../README.md), except using this repo instead of the `hello-world`:
```

Comment:
```suggestion
Then build this extension and launch it — this largely follows from the instructions in the top-level [README](../README.md), except using this example instead of the `hello-world` one:
```